src/cli/mod.rs

```diff
@@ -177,7 +185,7 @@ impl SudoOptions {
             self.action = SudoAction::ResetTimestamp;
         } else if self.validate {
             self.action = SudoAction::Validate;
-        } else if self.list {
+        } else if !matches!(self.list, List::None) {
```

Comment:
maybe using `self.list != List::None` is clearer, given that  `List` implements `PartialEq`